code/modules/vending/clothesmate.dm

```diff
@@ -4,7 +4,7 @@
 	desc = "A vending machine for clothing."
 	icon_state = "clothes"
 	icon_deny = "clothes-deny"
-	product_slogans = "Dress for success!;Prepare to look swagalicious!;Look at all this free swag!;Why leave style up to fate? Use the ClothesMate!"
+	product_slogans = "Dress for success!;Prepare to look swagalicious!;Look at all this not free swag!;Why leave style up to fate? Use the ClothesMate!"
```

Comment:
```suggestion
	product_slogans = "Dress for success!;Prepare to look swagalicious!;Look at all this swag!;Why leave style up to fate? Use the ClothesMate!"
```

Reply:
Tbh just change it to that

Reply:
I lol'd

Reply:
I think the not free really adds to the immersion though, really makes it a better line in general

Reply:
I lol'd _x2_